History.md

```diff
@@ -1,5 +1,18 @@
 # Liquid Change Log
 
+## 5.0.1 / 2021-03-24
+
+### Fixes
+* Add ParseTreeVisitor to Echo tag (#1414) [CP Clermont]
+* Fix Rubocop (#1402) [Dylan Thacker-Smith]
+* Add a hint to sign CLA only if it's the first time (#1406, #1405) [ADTC]
+* Test with ruby 3.0 as the latest ruby version (#1398) [Dylan Thacker-Smith]
+* Handle carriage return in newlines_to_br (#1391) [Unending]
+* Build the tokenizer through the parse context for liquid-c (#1386) [Dylan Thacker-Smith]
```

Comment:
This isn't really a fix.  It is more of an internal refactor.  I think we could just remove it and have liquid-c 5.0 depend on liquid version `>= 5.0.1`

```suggestion
```